include/GafferImage/ImageReader.h

```diff
@@ -110,10 +110,18 @@ class ImageReader : public ImageNode
 		Gaffer::StringPlug *colorSpacePlug();
 		const Gaffer::StringPlug *colorSpacePlug() const;
 
+		Gaffer::StringPlug *colorSpaceMetadataNamePlug();
```

Comment:
OK, here we go : https://github.com/GafferHQ/gaffer/pull/2087.

Lucien, once this is merged, could you rebase and confirm that the simple `compoundData->writable()["metadata"] = imagePlug->metadataPlug()->getValue();` approach works? I know this one is dragging on, so give me a shout if you want a hand getting it over the finish line...